code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -3999,6 +4035,101 @@ var/global/list/belly_colorable_only_fullscreens = list("a_synth_flesh_mono",
+				if(!isnull(item_multiplier_input))
+					hhost.vore_selected.item_multiplier = CLAMP(item_multiplier_input, 0.1, 10)
+					hhost.update_fullness()
+				. = TRUE
+		if("b_health_impacts_size")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				hhost.vore_selected.health_impacts_size = !hhost.vore_selected.health_impacts_size
+				hhost.update_fullness()
+				. = TRUE
+		if("b_resist_animation")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				hhost.vore_selected.resist_triggers_animation = !hhost.vore_selected.resist_triggers_animation
+				. = TRUE
+		if("b_size_factor_sprites")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				var/size_factor_input = input(user, "Set the impact all belly content's collective size has on your vore sprite. 1 means no scaling, 0.5 means content counts half as much, 2 means contents count double. (Range from 0.1 - 3)", "Size Factor") as num|null
+				if(!isnull(size_factor_input))
+					hhost.vore_selected.size_factor_for_sprite = CLAMP(size_factor_input, 0.1, 3)
+					hhost.update_fullness()
+				. = TRUE
+		if("b_tail_to_change_to")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				var/tail_choice = tgui_input_list(usr, "Which tail sprite do you want to use when your [lowertext(host.vore_selected.name)] is filled?","Select Sprite", global.tail_styles_list)
+				if(!tail_choice) //They cancelled, no changes
+					return FALSE
+				else
+					hhost.vore_selected.tail_to_change_to = tail_choice
+				. = TRUE
+		if("b_tail_color")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				var/newcolor = input(usr, "Choose tail color.", "", hhost.vore_selected.tail_colouration) as color|null
+				if(newcolor)
+					hhost.vore_selected.tail_colouration = newcolor
+				. = TRUE
+		if("b_tail_color2")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				var/newcolor = input(usr, "Choose tail secondary color.", "", hhost.vore_selected.tail_extra_overlay) as color|null
+				if(newcolor)
+					hhost.vore_selected.tail_extra_overlay = newcolor
+				. = TRUE
+		if("b_tail_color3")
+			if (istype(host, /mob/living/carbon/human))
+				var/mob/living/carbon/human/hhost = host
+				var/newcolor = input(usr, "Choose tail tertiary color.", "", hhost.vore_selected.tail_extra_overlay2) as color|null
+				if(newcolor)
+					hhost.vore_selected.tail_extra_overlay2 = newcolor
+				. = TRUE
+>>>>>>> 8ff0932889... Adds Toggleable Stomachs w/ toggleable visibility (#16052)
```

Comment:
deconf, we likely already have them, so they might be twice now

Reply:
Also shouldn't be human exclusive.